contracts/CollateralTracker.sol

```diff
@@ -1542,7 +1542,7 @@ contract CollateralTracker is ERC20Minimal, Multicall {
         uint256 numLegs = tokenId.countLegs();
 
         unchecked {
-            for (uint256 index = 0; index < numLegs; ++index) {
+            for (uint256 index; index < numLegs; ++index) {
```

Comment:
Please revert. Although this is logically identical, we explicitly specify =0 here for clarity.